checkstyle.sh

```diff
@@ -12,20 +8,4 @@ if ! flake8; then
     exit_code=1
 fi
 
-# Find files which use the unicode type but (heuristically) don't make it py3
-# safe
```

Comment:
What would you say to making this into a check for using `unicode` _at all_? It should all be `str` now in py3-only code.

Reply:
After thinking about it, I don't think it's important. Who will use `unicode` instead of ` str`? There is no `unicode` built-in anymore, and I don't see someone using it with no one noticing in review. I'm more worried of having yet another tool/script that could generate false positives, require maintenance, dependencies, etc.

Reply:
True, I suppose; this only checks _our_ code (core) anyway, and neither of us is going to approve any new patches that override or replace `str`. I'll just give this branch a once-over for any use of `unicode` that's not in a comment.